conformance/local_symtab_imports.ion

```diff
@@ -0,0 +1,152 @@
+  // The import would cause an error if import were attempted.
+  (toplevel $ion_symbol_table::{symbols:["a"], imports:({name:"not-in-catalog"})}
+            '#$10')
+  (produces a))
+
+
+// Non-exact-match imports
+
+(ion_1_x "If no max_id, lack of exact-match must raise an error"
+  (then (toplevel $ion_symbol_table::{imports:[{name:"not-in-catalog"}]})
+        (signals "No exact-match for not-in-catalog@1"))
+  (then (toplevel $ion_symbol_table::{imports:[{name:"empty", version:2}]})
+        (signals "No exact-match for empty@2")))
+
+(ion_1_x "If max_id not non-negative int, lack of exact-match must raise an error"
+  (then (toplevel $ion_symbol_table::{imports:[{name:"not-in-catalog", max_id:null.int}]})
+        (signals "No exact-match for not-in-catalog@1"))
+  (then (toplevel $ion_symbol_table::{imports:[{name:"empty", version:2, max_id:-5}]})
+        (signals "No exact-match for empty@2")))
+
+
+(ion_1_0 "When max_id is valid, pad/truncade mismatched or absent SSTs"
+  (then (toplevel $ion_symbol_table::{imports:[{name:"absent", max_id:0}, {name:"abcs"}]}
+                  '#$10')
+        (produces a))
+  (then (toplevel $ion_symbol_table::{imports:[{name:"absent", max_id:2}, {name:"abcs"}]}
+                  '#$10' '#$11' '#$12')
+        (denotes (symbol (absent "absent" 1)) (symbol (absent "absent" 2)) (symbol "a")))
+  (then (toplevel $ion_symbol_table::{imports:[{name:"empty", version:2, max_id:3},
+                                               {name:"abcs"}]}
+                  '#$10' '#$11' '#$12' '#$13')
+        (denotes (symbol (absent "empty" 1))
+                 (symbol (absent "empty" 2))
+                 (symbol (absent "empty" 3))
+                 (symbol "a")))
+  (then (toplevel $ion_symbol_table::{imports:[{name:"mnop", version:2, max_id:2},
+                                               {name:"abcs"}]}
+                  '#$10' '#$11' '#$12')
+        // mnop v2 doesn't exist, so we should get truncated v4, which has a gap
+        (denotes (symbol (absent "mnop" 1)) (symbol "n") (symbol "a"))))
+
+
+(ion_1_0 "Exact-match imports"
+  (each (toplevel $ion_symbol_table::{imports:[{name:"abcs"           }]})
+        (toplevel $ion_symbol_table::{imports:[{name:"abcs", version:1}]})
+        (then (toplevel '#$10')
+              (produces a))
+        (then (toplevel '#$11')
+              (signals "Invalid symbol address: 11")))
+  (then (toplevel $ion_symbol_table::{imports:[{name:"abcs", version:2}]})
+        (then (toplevel '#$10' '#$11')
+              (produces a b))
+        (then (toplevel '#$12')
+              (signals "Invalid symbol address: 12"))))
+
+
+(ion_1_0 "Exact-match with larger max_id inserts padding"
+  (toplevel $ion_symbol_table::{imports:[{name:"empty", version:1, max_id:3},
+                                         {name:"abcs"}]}
+            '#$13')
+  (produces a))
+
+
+// Multiple imports
+
+// TODO:[TAB-BOUNDS] This should be easier.
```

Comment:
What is `TAB-BOUNDS`?

Reply:
Oh, that's a label so I can find the test case when/if we add an expectation (to be proposed) for checking symbol table boundaries.  I've got a ton of those in the semantic model.

If you'd prefer, I can cut a placeholder issue and link to that instead; I just find this a nice lightweight mechanism.